scripts/gen_openapi_server.sh

```diff
@@ -1,28 +1,28 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -e
 
 echo "Generating the OpenAPI server"
 
-ROOT_FOLDER="${ROOT_FOLDER:-..}"
+PROJECT_ROOT=$(dirname "$(realpath "$0")")/..
```

Comment:
what are your thought about using this as an absolute path, to make it consistent with https://github.com/kubeflow/model-registry/pull/256/files ?

Reply:
good catch!